src/Assets/SteamVR_Input.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
The input controls are configured in the `feature/#10-create-dial-control` branch. The `SteamVR_Input` folder and `ActionSetClasses.meta` file should not be included in this PR, these files will overwrite the files in the dial control branch.